src/main/resources/META-INF/mods.toml

```diff
@@ -19,20 +19,20 @@ modId="minecolonies" #mandatory
 # The version number of the mod - there's a few well known ${} variables useable here or just hardcode it
 version="${file.jarVersion}" #mandatory
  # A display name for the mod
-displayName="Minecolonies" #mandatory
+displayName="MineColonies" #mandatory
 # A URL to query for updates for this mod. See the JSON update specification <here>
 #updateJSONURL="http://myurl.me/" #optional
 # A URL for the "homepage" for this mod, displayed in the mod UI
-# displayURL="http://example.com/" #optional
+displayURL="https://minecolonies.com" #optional
 # A file name (in the root of the mod JAR) containing a logo for display
-# logoFile="structurize.png" #optional
+logoFile="images/squarelogo.png" #optional
```

Comment:
This probably won't work as intended

Reply:
*loads minecraft and tests* Oh wow. I definitely did *not* intend for it to crash.

Reply:
Okay, now it doesn't crash, but the logo doesn't show up either.